elements/dash-accordion/dash-accordion.e2etest.js

```diff
@@ -0,0 +1,113 @@
+const helper = require('../../tools/selenium-helper.js');
+const expect = require('chai').expect;
+const {Key, By} = require('selenium-webdriver');
+
+describe('dash-accordion', function() {
+  beforeEach(function() {
+    return this.driver.get(`${this.address}/dash-accordion_demo.html`)
+      .then(_ => helper.waitForElement(this.driver, 'dash-accordion'));
+  });
+
+  it('should focus the next heading on [arrow right]',
+    async function() {
```

Comment:
It seems like every tests needs to do this initial bit of boilerplate, would it be possible to package it up into a reusable function?